samples/__tests__/e2e.test.ts

```diff
@@ -65,8 +65,12 @@ describe('with mediator', () => {
     const bobKeyAtBobMediator = bobInboundConnection?.verkey
     logger.test('bobInboundConnection', bobInboundConnection)
     */
-    const aliceMediatorConnection: ConnectionRecord | undefined = await aliceAgent.mediationRecipient.getDefaultMediatorConnection()
-    const bobMediatorConnection: ConnectionRecord | undefined = await bobAgent.mediationRecipient.getDefaultMediatorConnection()
+    const aliceMediatorConnection:
```

Comment:
Some weird formatting happening here?

Reply:
that is strange. I removed the type. it will now look something like.
```
    const aliceMediatorConnection = await aliceAgent.mediationRecipient.getDefaultMediatorConnection()
    const bobMediatorConnection = await bobAgent.mediationRecipient.getDefaultMediatorConnection()
```